README.md

```diff
@@ -1 +1,105 @@
-# drone-artifactory-docker-buildinfo
\ No newline at end of file
+# Artifactory Docker Build Info Plugin
+
+A Drone plugin that publishes Docker image build information to JFrog Artifactory, including build details, VCS information, and SHA256 verification.
+
+## Features
+
+- Publishes Docker image build information to JFrog Artifactory
```

Comment:
we don't need to mention all this info, instead just have the synopsis, supported plugin image arch